src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/RequiredReferenceInputArgumentCollection.java

```diff
@@ -11,8 +11,8 @@
 public final class RequiredReferenceInputArgumentCollection extends ReferenceInputArgumentCollection {
     private static final long serialVersionUID = 1L;
 
-    @WorkflowResource(input=true, output=false, companionResources = {"referenceDictionary", "referenceIndex"})
     @Argument(fullName = StandardArgumentDefinitions.REFERENCE_LONG_NAME, shortName = StandardArgumentDefinitions.REFERENCE_SHORT_NAME, doc = "Reference sequence file", optional = false)
+    @WorkflowResource(input=true, output=false, companionResources = {"referenceDictionary", "referenceIndex"})
```

Comment:
The reference could potentially have two indices (`.fai` and `.gzi`)

(This comment applies to `OptionalReferenceInputArgumentCollection` as well)

Reply:
Yeah, optional companions are in the Barclay branch as well.